mypy.ini

```diff
@@ -76,7 +76,6 @@ exclude = (?x)
    |tests/storage/test_base.py
    |tests/storage/test_roommember.py
    |tests/test_metrics.py
-   |tests/test_phone_home.py
    |tests/test_server.py
    |tests/test_state.py
    |tests/test_terms_auth.py
```

Comment:
I'm guessing the other three test files this change need additional work to pass mypy?

Reply:
`tests/storage/test_monthly_active_users.py` and `tests/storage/test__base.py` are not in exclusion list and is different to `tests/storage/test_base.py`. There is a `_` more.

There a more errors:
```
tests/storage/test_roommember.py:37: error: Argument 3 of "prepare" is incompatible with supertype "HomeserverTestCase"; supertype defines the argument type as "HomeServer"  [override]
```
I am do not understand why here is used `HomeserverTestCase`.

Reply:
Thanks for explaining that. We can deal with those problems further down the line.